packages/js/core/src/uri-resolution/resolvers/api/ApiAggregatorResolver.ts

```diff
@@ -10,16 +10,24 @@ import { CreateApiFunc } from "./types/CreateApiFunc";
 import { UriResolutionResult } from "../../core/types/UriResolutionResult";
 import { UriToApiResolver, UriResolutionStack } from "../../core";
 import { ApiResolver } from "./ApiResolver";
+import { Queue } from "../../../utils/Queue";
 
 export type ApiAggregatorResolverResult = UriResolutionResult & {
   resolverUri?: Uri;
 };
 
 export class ApiAggregatorResolver implements UriToApiResolver {
+  private hasLoadedAllResolvers: boolean;
+
   constructor(
     private readonly createApi: CreateApiFunc,
-    private deserializeOptions?: DeserializeManifestOptions
-  ) {}
+    private deserializeOptions?: DeserializeManifestOptions,
+    disablePreloadResolvers?: boolean
+  ) {
+    if (disablePreloadResolvers) {
```

Comment:
This is a sort of hacky solution to the problem of using the aggregator in core-js tests

Reply:
hmmm, should we use a `TEST` environment variable maybe?

Reply:
Maybe, but I think this is also has usecases elsewhere, eg a performance increase if you know all of your resolvers are local